### DISCUSSION TITLE
[Arch Linux] Xfwm4 settings utilities not showing up in the Application Menu

### DISCUSSION THREAD

**Discussion by isf63** [lxqt/lxqt#2473] [Q&A] [answered]
There are several settings utilities that ship with Xfwm4. These are their corresponding .desktop files:

```
[ applications]$ ls $PWD/xfce*
/usr/share/applications/xfce4-about.desktop       /usr/share/applications/xfce-wmtweaks-settings.desktop
/usr/share/applications/xfce-wm-settings.desktop  /usr/share/applications/xfce-workspaces-settings.desktop
```

They have categories that should show up in the Application Menus, namely 'Utility' and 'Settings':

```
[ applications]$ grep Categories= $PWD/xfce*
/usr/share/applications/xfce4-about.desktop:Categories=Utility;X-XFCE;X-Xfce-Toplevel;
/usr/share/applications/xfce-wm-settings.desktop:Categories=XFCE;GTK;Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-PersonalSettings;
/usr/share/applications/xfce-wmtweaks-settings.desktop:Categories=XFCE;GTK;Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-PersonalSettings;
/usr/share/applications/xfce-workspaces-settings.desktop:Categories=XFCE;GTK;Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-PersonalSettings;
```

As far as I can tell, the Settings category is handled by `lxqt-config.menu` which is included by other menus. Utility is handled directly by the various `lxqt-applications ...` menus.

<hr>

It's unknown why they are not included in my start menus. In the past I manually added them, but since forgot how to.

**Comment by tsujan** [comment, marked as answer]
Search for `OnlyShowIn` and....

**Reply by isf63** [reply]
Thanks. Commented out `OnlyShowIn=XFCE;` and restarted the panel

**Reply by tsujan** [reply]
You could (1) copy them to your home, (2) remove `OnlyShowIn=XFCE;`, (3) copy them to `~/.local/share/applications/`, and wait a few seconds.

**Reply by stefonarch** [reply]
Wiki: https://github.com/lxqt/lxqt/wiki/ConfigWindowManagers#xfwm4

**Reply by isf63** [reply]
@stefonarch Thanks. I searched the Wiki but for "Xfwm" so didn't find that.
